<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1153</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:url(:/icons/wepik-photo-mode-2022923-115059.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>811</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2px solid#9708B4;
    border-bottom-color: #9708B4; /* same as the pane color */
    border-top-left-radius: 6px;
    border-top-right-radius: 6px;
    min-width:60ex;
    padding: 2px;
     font: bold, &quot;Comic Sans MS&quot;;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #085394, stop: 0.4 #9708B4,
                                stop: 0.5 #9708B4, stop: 1.0 #9708B4);
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #9708B4; /* same as pane color */
}
QLabel{
background:transparent;
font:Century gothic;
font-size:20px;
color:#8b8b8b;
}

QDateEdit{
background:#8b8b8b;
}
QGroupBox{
font-color:rgb(255, 255, 255);
}



</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QLineEdit" name="l_eventname">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072; 
font:Century gothic;
font-size:24px;
}
QDateEdit{
background:transparent;
border:none;
}
</string>
      </property>
      <property name="text">
       <string>Event_name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="l_eventype">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072; 
font:Century gothic;
font-size:24px;
}
QDateEdit{
background:transparent;
border:none;
}
</string>
      </property>
      <property name="text">
       <string>Event_type</string>
      </property>
     </widget>
     <widget class="QToolButton" name="PB_ajout">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>50</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#01166B;
border-radius:60px;
}
QToolButton{
background:#01166B;
border-radius:60px;} 

QToolButton::hover{
background:#9708B4;
border-radius:60px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/icons8-ajouter-64.png</normaloff>:/icons/icons8-ajouter-64.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>190</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#01166B;
border-radius:60px;
}
QToolButton{
background:#01166B;
border-radius:60px;} 

QToolButton::hover{
background:#9708B4;
border-radius:60px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/icons8-effacer-64.png</normaloff>:/icons/icons8-effacer-64.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="l_dates">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072; 
font:Century gothic;
font-size:24px;
}
QDateEdit{
background:transparent;
border:none;
}
</string>
      </property>
      <property name="text">
       <string>Dates</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Modifier/Supprimer</string>
     </attribute>
     <widget class="QLineEdit" name="l_eventname_supp">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>60</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072; 
font:Century gothic;
font-size:24px;
}
QDateEdit{
background:transparent;
border:none;
}
</string>
      </property>
      <property name="text">
       <string>Event_name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="l_eventtype_modif">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072; 
font:Century gothic;
font-size:24px;
}
QDateEdit{
background:transparent;
border:none;
}
</string>
      </property>
      <property name="text">
       <string>Event_type</string>
      </property>
     </widget>
     <widget class="QToolButton" name="PB_modif">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>350</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#01166B;
border-radius:60px;
}
QToolButton{
background:#01166B;
border-radius:60px;} 

QToolButton::hover{
background:#9708B4;
border-radius:60px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/icons8-engrenage-64.png</normaloff>:/icons/icons8-engrenage-64.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="pb_supp">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>150</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#01166B;
border-radius:60px;
}
QToolButton{
background:#01166B;
border-radius:60px;} 

QToolButton::hover{
background:#9708B4;
border-radius:60px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/icons8-effacer-64 (1).png</normaloff>:/icons/icons8-effacer-64 (1).png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="l_Dates_modif">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072; 
font:Century gothic;
font-size:24px;
}
QDateEdit{
background:transparent;
border:none;
}
</string>
      </property>
      <property name="text">
       <string>Dates</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="l_eventname_modif">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072; 
font:Century gothic;
font-size:24px;
}
QDateEdit{
background:transparent;
border:none;
}
</string>
      </property>
      <property name="text">
       <string>Event_name</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QTableView" name="tab_event">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>721</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072; 
font:Century gothic;
font-size:24px;
}
QDateEdit{
background:transparent;
border:none;
}
</string>
      </property>
      <property name="text">
       <string>Research</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#01166B;
border-radius:60px;
}
QToolButton{
background:#01166B;
border-radius:60px;} 

QToolButton::hover{
background:#9708B4;
border-radius:60px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Downloads/chercher.png</normaloff>:/Downloads/chercher.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_6">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>30</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#01166B;
border-radius:60px;
}
QToolButton{
background:#01166B;
border-radius:60px;} 

QToolButton::hover{
background:#9708B4;
border-radius:60px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Downloads/pdf.png</normaloff>:/Downloads/pdf.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Annulation/Reportation</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>291</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{
color:rgb(255, 255, 255);
}</string>
      </property>
      <property name="title">
       <string>Annulation</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>40</y>
        <width>301</width>
        <height>341</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{
color:rgb(255, 255, 255);
}
</string>
      </property>
      <property name="title">
       <string>Reportation</string>
      </property>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>271</width>
         <height>191</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QCalendarWidget QToolButton {

  	color: white;
  	font-size: 21px;
  	icon-size:30px, 30px;
  	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #cccccc, stop: 1 #333333);
  }
  QCalendarWidget QMenu {
 
  	color: white;
  	font-size: 10px;
  	background-color: rgb(100, 100, 100);
  }
  QCalendarWidget QSpinBox { 
  
  	color: white; 
  	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #cccccc, stop: 1 #333333); 
  	selection-background-color: rgb(136, 136, 136);
  	selection-color: rgb(255, 255, 255);
  }
  QCalendarWidget QSpinBox::up-button { subcontrol-origin: border;  subcontrol-position: top right;  width:30px; }
  QCalendarWidget QSpinBox::down-button {subcontrol-origin: border; subcontrol-position: bottom right;  width:65px;}
  QCalendarWidget QSpinBox::up-arrow { width:30px;  height:30px; }
  QCalendarWidget QSpinBox::down-arrow { width:30px;  height:30px; }
   
  /* header row */
  QCalendarWidget QWidget { alternate-background-color: rgb(128, 128, 128); }
   
  /* normal days */
  QCalendarWidget QAbstractItemView:enabled 
  {

  	color: rgb(180, 180, 180);  
  	background-color: black;  
  	selection-background-color: rgb(64, 64, 64); 
  	selection-color: rgb(0, 255, 0); 
  }
   
  /* days in other months */
  /* navigation bar */
QCalendarWidget QWidget#qt_calendar_navigationbar
{ 
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #cccccc, stop: 1 #333333); 
}

QCalendarWidget QAbstractItemView:disabled 
{ 
color: rgb(64, 64, 64); 
}
</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1153</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
